### PR TITLE
[NT-973] Fix payment icon error

### PR DESCRIPTION
**Pull request by cdolm92**
# 📲 What

Backers can now see a fix icon next to the payment method that errored.

# 🤔 Why

This work enables a backer with an errored pledge to know which card had the issue so that they can retry it.

# 🛠 How

Right now we display the icon based on whether the backing errored and if the card is selected. In the upcoming PR we will change the pledge view context so that the fix icon is not displayed by whether or not the card is selected. 

# 👀 See
Only the card that errors shows the icon.

Screen Shot 2020-03-11 at 1 55 52 PM
Screen Shot 2020-03-11 at 1 56 23 PM

# ✅ Acceptance criteria

Test w/ Charles
- [x] Use breakpoint on the project page on Charles and navigate to a backed project on the simulator.  Change the backing status to `errored` on Charles. Navigate through the fix pledge flow and you should see the errored card

### CODE REVIEW

File: Library/ViewModels/ProjectPamphletViewModelTests.swift [hunk 1]
```diff
@@ -460,43 +460,7 @@ final class ProjectPamphletViewModelTests: TestCase {
     }
   }
 
-  func testGoToDeprecatedManagePledge_ManagingPledge_featureNativeCheckoutPledgeView_Disabled() {
```
Comment (ifbarrera) [inline Library/ViewModels/ProjectPamphletViewModelTests.swift]: Did this get missed when we removed the old checkout code, or does this PR need to be updated with the latest changes from master?

File: Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift [hunk 2]
```diff
@@ -54,7 +69,14 @@ final class PledgeCreditCardView: UIView {
     _ = (self.containerStackView, self)
       |> ksr_addSubviewToParent()
 
-    _ = ([self.lastFourLabel, self.expirationDateLabel], self.labelsStackView)
+    _ = (self.fixIconImageView, self.fixIconView)
```
[the remark this answers is not in the document]
Comment (cdolm92) [reply Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift]: It was easier to have it match the design this way.

File: Library/ViewModels/ProjectPamphletViewModelTests.swift (hunk 1 above)
Comment (cdolm92) [reply Library/ViewModels/ProjectPamphletViewModelTests.swift]: I'm updating this PR with the changes in `featured-errored-pledges`. `featured-errored-pledges` should be updated with the latest changes from `master`.

**Comment by nativeksr** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Big PR</td>
    </tr>
  </tbody>
</table>

### SwiftFormat found issues:

| File | Rules |
| ---- | ----- |
| Kickstarter-iOS/Views/Cells/PledgeCreditCardView.swift | blankLinesBetweenScopes |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Pull request merged**